packages/nimble-components/docs/component-usage-patterns/combobox-select-menu.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story } from '@storybook/blocks';
+import * as selectStories from '../../src/select/tests/select.stories';
+import * as comboboxStories from '../../src/combobox/tests/combobox.stories';
+import * as menuButtonStories from '../../src/menu-button/tests/menu-button.stories';
+
+<Meta title="Usage/Combobox, Select, & Menu Button"/>
+
+# Combobox, Select, & Menu Button Usage
```

Comment:
I agree that it would likely be the best option to have one block with this type of information that can be used in multiple component pages. It seems too hyper-specific for a high-level page, and we should avoid creating new pages for every little detail. The component definitions do not need to be here. Put the component definitions in the component documentation, and have a section/block in each to the effect of "is this the right control?" and single-sentence answers for select, combobox, and menu-button.